src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/læremidler/domain/Studienivå.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.tilleggsstonader.sak.vedtak.læremidler.domain
+
+enum class Studienivå {
+    VIDEREGÅENDE,
```

Comment:
Denne eksisterte fra før. Kan godt endre til det du har i frontend, @sarahjelle 
```suggestion
    VIDEREGÅENDE_UTDANNING,
```